### PR TITLE
Update blog.md

### PR DESCRIPTION
**Pull request by vpavlin**
Hey! Cool blog! I went over it and quickly fixed some typos and tried to rephrase where I got lost. Let me know WDYT:)

### CODE REVIEW

File: docs/blog.md [hunk 1]
```diff
@@ -270,7 +270,7 @@ The `subscribe` method returns the `unsubscribe` method, but in my case I segreg
 
 #### Sender
 
-Here is where the polling runs and does batch calling to all the token contracts and sends to the waku topic that we reated.
+Here is where the polling runs and does batch calling to all the token contracts and sends to the Waku topic that we ??reated??.
```
Comment (vpavlin) [inline docs/blog.md]: I was not sure what `reated` means or which word was it supposed to be:)

File: docs/blog.md [hunk 2]
```diff
@@ -404,11 +404,11 @@ Now we had our backend polling logic and waku setup ready, lets import these fun
     }
 ```
 
-You will notice the code blocks above uses other state values like wakuNodeStatus, lastSynced these are metrics to show user the peer connection status and last received time of a waku message.
+You will notice the code blocks above uses other state values like `wakuNodeStatus` or `lastSynced` these are metrics to show the user the peer connection status and last time we received a message ove Waku.
 
 #### Wrap
 
-Although the problem that I solved here can be implmented many ways, having decentralized solutions makes your app fully decentralized. Waku can be applied in several other cases too.
+Although the problem that I solved here can be implmented in many ways, having decentralized solutions makes your app fully decentralized. Waku can be applied in several other cases too.
```
Comment (vpavlin) [inline docs/blog.md]: Would you mind talking a bit more about Waku here? Maybe explain the decentralization and how Waku helps with it?

File: docs/blog.md (hunk 1 above)
Comment (5war00p) [reply docs/blog.md]: 😅 its `created`.

File: docs/blog.md [hunk 3]
```diff
@@ -404,11 +404,11 @@ Now we had our backend polling logic and waku setup ready, lets import these fun
...
+You will notice the code blocks above uses other state values like `wakuNodeStatus` or `lastSynced` these are metrics to show the user the peer connection status and last time we received a message ove Waku.
 
 #### Wrap
 
-Although the problem that I solved here can be implmented many ways, having decentralized solutions makes your app fully decentralized. Waku can be applied in several other cases too.
+And it's a wrap. Some of you may yell at screen that this problem can be solved in other ways. Yes it is, but the goal over here is to show how Waku plays key role in DApps irrespective of the size of the application. Additionally, users of DApp's may look for complete decentralization, if it misses then companies may loose their customers. So, to keep fully decentralize setup for your DApp one of the important characteristic is decentralized communication. By integrating Waku, it enables decentralised communication features to your application without compromising security or privacy or may be scalability if you are integrating in large applications.
```
Comment (5war00p) [inline docs/blog.md:411]: @vpavlin Updated conclusion. Other changes looks fine. Let me know thoughts on this summary.

**Pull request merged**